app/src/main/java/swati4star/createpdf/util/CommonCodeUtils.java

```diff
@@ -71,4 +72,12 @@ public static void updateView(Activity mActivity, int imageCount, ArrayList<Stri
         mCreatedImages.setAdapter(extractImagesAdapter);
         mCreatedImages.addItemDecoration(new ViewFilesDividerItemDecoration(mActivity));
     }
+
+    public static void closeBottomSheetUtil(BottomSheetBehavior sheetBehavior) {
+        if (sheetBehavior.getState() == BottomSheetBehavior.STATE_EXPANDED)
+            sheetBehavior.setState(BottomSheetBehavior.STATE_COLLAPSED);
+    }
+    public static boolean checkSheetBehaviourUtil(BottomSheetBehavior sheetBehavior) {
+        return sheetBehavior.getState() == BottomSheetBehavior.STATE_EXPANDED;
+    }
```

Comment:
Line 77and function `checkSheetBehaviourUtil()` are the same.

Reply:
we need  boolean value to know whether the sheet is expanded or closed in main activity. and the above function(closeBottomSheetUtil) just closes the expanded sheet but doesn't return any value. in the main activity we need to know the current state of bottomsheet and then act accordingly on backpressed. So two separate methods are declared

Reply:
Instead of 
```
 if (sheetBehavior.getState() == BottomSheetBehavior.STATE_EXPANDED)
            sheetBehavior.setState(BottomSheetBehavior.STATE_COLLAPSED);
```

you can use 
```
 if (checkSheetBehaviourUtil())
            sheetBehavior.setState(BottomSheetBehavior.STATE_COLLAPSED);
```

Reply:
Also write a Javadoc style comment for both the functions in `CommonCodeUtils` class just like there are comments in the same class for other functions.

Reply:
Done. Thanks for the co-operation and guidance